__tests__/app.test.js

```diff
@@ -31,3 +31,46 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test.only("responds with an article object of given id", () => {
```

Comment:
```suggestion
  test("responds with an article object of given id", () => {
```

Try not to commit the `.only`'s, it's a really easy way to introduce bugs 😁